tests/feedback.test.ts

```diff
@@ -73,127 +114,353 @@ describe('feedback submission', () => {
   });
 
   test('admins can view feedback from any member', async () => {
+    const event = EventFactory.fake();
```

Comment:
I removed your other function that was creating the event–we have `EventFactory.fake()` to do that for us.